packages/block-library/src/cover/edit.js

```diff
@@ -134,6 +161,8 @@ class CoverEdit extends Component {
 					{}
 				),
 			} );
+
+			this.toggleIsEditing();
```

Comment:
Same comment as above here. I think this should specifically set `isEditing` to `false` instead of just toggling.